.env

```diff
@@ -0,0 +1,6 @@
+# database
```

Comment:
new .env file that describes the db to stand up locally.

we can also add the s3 filepaths in here to use with our lambda scripts if we want.